packages/@react-spectrum/radio/docs/Radio.mdx

```diff
@@ -138,76 +157,67 @@ Implement your own validation logic in your app and pass `"invalid"` to the Radi
 
 ### Orientation
 
-A RadioGroup is vertically aligned by default.
-An optional prop can be passed to the RadioGroup to position the Radio elements inline.
+RadioGroups are vertically oriented by default.
+The `orientation` prop can be used to change the orientation to horizontal.
 
 ```tsx example
-<RadioGroup label="Example Options" orientation="horizontal">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
+<RadioGroup label="Favorite avatar" orientation="horizontal">
+  <Radio value="wizard">Wizard</Radio>
+  <Radio value="dragon">Dragon</Radio>
 </RadioGroup>
 ```
 
-### Label position
-
-The default position of a label is on top of the RadioGroup.
-An optional prop can be passed to the RadioGroup that will position the label at the start of the topmost Radio.
+### Label position and alignment
+[View guidelines](https://spectrum.adobe.com/page/radio-button/#Label)
 
-```tsx example
-<RadioGroup label="Example Options" labelPosition="side">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
-</RadioGroup>
-```
-
-### Label align
-
-Labels can be aligned to the "start" or the "end" of the RadioGroup.
+By default, the label is positioned above the RadioGroup.
+The `labelPosition` prop can be used to position the label to the side. The `labelAlign` prop can
+be used to align the label as "start" or "end".
 For left-to-right (LTR) languages, "start" refers to the left most edge of the RadioGroup
 and "end" refers to the right most edge. For right-to-left (RTL) languages, this is flipped.
 
 ```tsx example
-<RadioGroup label="Example Options" labelAlign="end">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
+<RadioGroup label="Favorite avatar" labelPosition="side" labelAlign="end">
+  <Radio value="wizard">Wizard</Radio>
+  <Radio value="dragon">Dragon</Radio>
 </RadioGroup>
 ```
 
 ### Disabled
-[View Guidelines](https://spectrum.adobe.com/page/radio-button/#Disabled)
+[View guidelines](https://spectrum.adobe.com/page/radio-button/#Disabled)
 
 ```tsx example
-<RadioGroup label="Example Options" isDisabled>
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon">Dragon</Radio>
+<RadioGroup label="Favorite avatar" isDisabled>
+  <Radio value="wizard">Wizard</Radio>
+  <Radio value="dragon">Dragon</Radio>
 </RadioGroup>
 ```
 
 ```tsx example
-<RadioGroup label="Example Options">
- <Radio value="wizard">Wizard</Radio>
- <Radio value="dragon" isDisabled>Dragon</Radio>
+<RadioGroup label="Favorite avatar">
+  <Radio value="wizard">Wizard</Radio>
+  <Radio value="dragon" isDisabled>Dragon</Radio>
 </RadioGroup>
 ```
 
 ### Read only
-
+[View guidelines](https://spectrum.adobe.com/page/radio-button/#Read-only)
```

Comment:
```suggestion
[View guidelines]

(https://spectrum.adobe.com/page/radio-button/#Read-only)
```

Reply:
why?

Reply:
Ah whoops, thought this was the same issue that I noted in other places where the guidelines link was on the same line as the description, can ignore this one